test/exercises/duplicates/forStudents/README

```diff
@@ -0,0 +1,179 @@
+Data Processing Hands-On Exercise: Finding Duplicate Files
+==========================================================
+
+This directory contains the framework for writing a Chapel program to find
+duplicate files.
+
+What is in this directory?
+**************************
+
+FileHashing.chpl:
+  This module defines:
+  * a type SHA256Hash supporting the usual comparison and assignment operators
+    as well as writeln
+  * a function computeFileHash that returns a path
+  * a function relativeRealPath used to normalize paths
+
+duplicates.chpl:
+  This is the code you'll edit for this exercise.
+
+example-files/
+  This directory stores some example files for testing your program.
+
+expected-output:
+  This file contains the expected output for your program when run on the
+  example-files directory.
+
+Hands-On Tasks
+**************
+
+Task 1: Getting Started
+-----------------------
+
+To get started with this project, compile 'duplicates.chpl'.  Note that this
+program uses the OpenSSL library to compute SHA256 hashes and might need
+additional compilation flags to find that library on your system.
+For example, on Mac OS X, you might `brew install openssl` and then set
+LDFLAGS and CPPFLAGS to convey the location of the library to the Chapel
+compiler:
+
+  export LDFLAGS="-L/usr/local/opt/openssl/lib"
+  export CPPFLAGS="-I/usr/local/opt/openssl/include"
+
+
+Try compiling 'duplicates.chpl' and running it with the example-files
+directory. It should (incorrectly) report that all the files are duplicates.
+
+Feel free to look over the provided source code in duplicates.chpl and
+FileHashing.chpl.
+
+Task 2: Compute Hashes to Detect Duplicates
+-------------------------------------------
+
+Your next task is to get it reporting only actual duplicates. To do so, you'll
+need to adjust the program to compute the hashes of the files.  To do so, edit
+`computeHashes` near the top of the program. You'll want to call the provided
+function `computeFileHash` from the FileHashing module.
```

Comment:
This might click quicker if we give clearer instructions about the array, e.g.

> The function should compute the hash for each path and store it in the hashAndPath array.

We could alternatively put this clue in the comments of the function.